README.md

```diff
@@ -11,6 +11,7 @@ Some steps must be taken on each host that is going to have home alert software
 2. Ensure that the host used to deploy the installation can ssh into each of the users of the target hosts via ssh keys.
 3. Install ansible
  * `$ sudo apt-get install ansible`
+4. Ensure the `python3-opencv` package is available via apt. If not, you will have to install OpenCV with python3 bindings via some other method.
```

Comment:
May be nit picky but really this note should go in its own header section. I don't think it's a necessary step to check for most users. Only in special cases now should one not have the right packages available, like when installing to some custom OS. Maybe we can just put this note as a comment in the config file?

Reply:
More I think the more I don't think this needs to me mentioned in the readme.

Reply:
Sounds good.